Language/Structure/Control Structure/switchCase.adoc

```diff
@@ -83,6 +82,29 @@ switch (var) {
 ----
 [%hardbreaks]
 
+
+[float]
+=== Notes and Warnings
+// Add useful notes, tips, caveat, known issues, and warnings about this Reference term
+If you need to declare a variable in a case, use curly brackets {} to define a scope within the case statement to avoid a "crosses initialization of" compiler error. 
```

Comment:
```suggestion
If you need to declare a variable for use in a case, use link:../../further-syntax/curlybraces[curly braces] (`{`, `}`) to define a scope within the case statement to avoid a "crosses initialization of" compiler error. 
```
- Make it a little bit more clear why the reader might declare a variable in a case.
- Use Arduino's preferred name for braces: https://www.arduino.cc/reference/en/language/structure/further-syntax/curlybraces/
- Link to ["curly braces" reference page](https://www.arduino.cc/reference/en/language/structure/further-syntax/curlybraces/)
- Improve formatting of braces symbols